src/components/MobileNavLinksMenu.js

```diff
@@ -0,0 +1,90 @@
+import React, {useState} from "react";
+import {Link} from "react-router-dom";
+import styled from "styled-components";
+import {HamburgerButton} from "./HamburgerButton";
+
+const MobileNavLinkWrapper = styled.div`
+  @media only screen and (min-width: 768px) {
+    display: none;
+  }
+  @media only screen and (max-width: 768px) {
+    display: flex;
+    flex-flow: column;
+    background-color: rgb(245, 244, 220);
+    position: fixed;
+    top: 0;
+    right: 0;
+    height: 100vh;
+    width: 250px;
+    padding-top: 3.5rem;
+  }
+`;
+
+const NavLink = styled(Link)`
+  text-decoration: none;
+  color: #919191;
```

Comment:
I would recommend we create a shared `theme` config that contains all these colors and their names/nicknames. That way we can import them across the project and be sure they're consistent.